app.json

```diff
@@ -1,8 +1,14 @@
 {
     "name": "Cog",
     "description": "A hackathon hardware check-out system by HackMIT",
-    "repository": "https://github.com/techx/cog",
+    "repository": "https://github.com/prototriangle/cog",
     "env": {
+        "OAUTH_ID": {
```

Comment:
If I remember these were environment variables before. What is the behaviour of the app now that these are in app.json? What does this change? Are the docs updated?